SWLOR.Game.Server/Enumeration/PlanetType.cs

```diff
@@ -70,6 +70,13 @@ public enum PlanetType
             800,
             true)]
         Dathomir = 64,
+        [Planet("Dantooine",
+    "Dantooine - ",
```

Comment:
Please fix the tabbing here. It should be lined up similar to the other planet entries.